api/v1beta2/awsmachine_types.go

```diff
@@ -286,9 +286,10 @@ type CloudInit struct {
 // For more information on Ignition configuration, see https://coreos.github.io/butane/specs/
 type Ignition struct {
 	// Version defines which version of Ignition will be used to generate bootstrap data.
+	// Defaults to `2.3` if storageType is set to `ClusterObjectStore`.
+	// It will be ignored if storageType is set to `UnencryptedUserData`.
```

Comment:
We could mention why we now ignore it if `UnencryptedUserData` (e.g. not relevant as the UnencryptedUserData already carry a manifest that may contain a different version, or something similar?)